PiwikTracker/PiwikTracker.swift

```diff
@@ -281,18 +283,70 @@ extension PiwikTracker {
         dimensions.append(dimension)
     }
     
+    /// Set a permanent custom dimension by value and index.
+    ///
+    /// This is a convenience alternative to set(dimension:) and calls the exact same functionality. Also, it is accessible from Objective-C.
+    ///
+    /// - Parameter value: The value for the new Custom Dimension
+    /// - Parameter forIndex: The index of the new Custom Dimension
+    @objc public func setDimension(_ value: String, forIndex index: Int) {
+        set(dimension: CustomDimension( index: index, value: value ));
+    }
+    
     /// Removes a previously set custom dimension.
     ///
     /// Use this method to remove a dimension that was set using the `set(value: String, forDimension index: Int)` method.
     ///
     /// - Parameter index: The index of the dimension.
-    public func remove(dimensionAtIndex index: Int) {
+    @objc public func remove(dimensionAtIndex index: Int) {
         dimensions = dimensions.filter({ dimension in
             dimension.index != index
         })
     }
 }
 
+
+extension PiwikTracker {
+    
+    internal func getDefaultCVars() -> [CustomVariable] {
```

Comment:
What do you think about moving this to the `CustomVariable` struct? Something like this:
```Swift
extension CustomVariable {
    static var defaultVariables: [CustomVariable] {
        // return ...
    }
}
```